app/views/unattended/provisioning_templates/snippet/ansible_tower_callback_service.erb

```diff
@@ -15,9 +26,9 @@ After=network-online.target
 [Service]
 Type=oneshot
 <% if host_param('ansible_extra_vars') -%>
-ExecStart=/usr/bin/curl -k -s -H 'Content-Type: application/json' --data '{"host_config_key":"<%= host_param('ansible_host_config_key') %>", "extra_vars": <%=host_param('ansible_extra_vars') %>}' https://<%= host_param('ansible_tower_fqdn') %>/api/v2/job_templates/<%= host_param('ansible_job_template_id') %>/callback/
+ExecStart=/usr/bin/curl -k -s -H 'Content-Type: application/json' --data '{"host_config_key":"<%= host_param('ansible_host_config_key') %>", "extra_vars": <%=host_param('ansible_extra_vars') %>}' https://<%= host_param('ansible_tower_fqdn') %>${api_base}<%= host_param('ansible_job_template_id') %>/callback/
```

Comment:
I tested this change and here `${api_base}` doesn't render at all, so not sure if it should be `<%= api_base %> or something else in this erb template